### PR TITLE
Add beautysh 3.11

### PR DESCRIPTION
**Pull request by bollwyvl**
[beautysh](https://github.com/bemeurer/beautysh) is a nice little `bash` formatter.

- [X] [PR](https://github.com/bemeurer/beautysh/pull/28) with license for upstream for sdist/wheels.
  - [x] merged (within minutes hooray!)

### CODE REVIEW

File: recipes/beautysh/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+about:
+  home: https://github.com/bemeurer/beautysh
+  license: MIT
+  license_family: MIT
+  license_file: src/LICENSE
+  summary: A Bash beautifier for the masses.
+
+extra:
+  recipe-maintainers:
+    - bollwyvl
```
Comment (bollwyvl) [inline recipes/beautysh/meta.yaml]: @conda-forge-linter looks fine to me...

**Comment by bollwyvl** [comment]
@conda-forge-admin, please lint

File: recipes/beautysh/meta.yaml (hunk 1 above)
Comment (bemeurer) [reply recipes/beautysh/meta.yaml]: @bollwyvl Try `recipe-maintainers: [bollywyvl]`

File: recipes/beautysh/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{% set name = "beautysh" %}
+{% set version = "3.10" %}
```
Comment (bemeurer) [inline recipes/beautysh/meta.yaml]: If you can, please bump up to version `3.11`, cheers!

**Comment by bemeurer** [comment]
@bollwyvl Please feel free to add me as a maintainer as well :)

**Comment by bollwyvl** [comment]
Huzzah!

**Comment by marcelotrevisani** [comment]
Looks good to me! I'm going to merge this!

Thanks for you effort to put this in ``conda-forge`` ! :)

**Pull request merged**